### ISSUE TITLE
[bug]: Migration of db fails, 64-bit postgres cannot read db files generated by 32-bit postgres

### ISSUE THREAD

**Issue by chrisguida** [Start9Labs/mastodon-wrapper#10]
```
2022-01-13T16:08:08.444960Z 2022-01-13 16:08:08.443 UTC [108] FATAL:  database files are incompatible with server
2022-01-13T16:08:08.445399Z 2022-01-13 16:08:08.443 UTC [108] DETAIL:  The data directory was initialized by PostgreSQL version 12, which is not compatible with this version 13.5.
```

Probably need to upgrade Postgres to match the new 030 version of Mastodon.

**Comment by ProofOfKeags** [comment]
It actually seems like it needs to be *downgraded*. I'm weirded out though that postgres doesn't have the ability to migrate from 12 to 13.

**Comment by elvece** [comment]
Was the version of mastodon updated from what we had on 02x?

**Comment by chrisguida** [comment]
@elvece yes, the old version of mastodon was no longer building because some dependency got yanked

**Comment by chrisguida** [comment]
Even after downgrading postgres, a similar bug occurred because 64-bit postgres cannot read 32-bit postgres db files. Renaming to reflect

**Comment by chrisguida** [comment]
Fixed by #12
